support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/config/CassandraProperties.java

```diff
@@ -0,0 +1,80 @@
+package org.apereo.cas.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * This is {@link CassandraProperties}.
+ *
+ * @author David Rodriguez
+ * @since 5.1.0
+ */
+@ConfigurationProperties("cassandra")
+public class CassandraProperties {
+
+    private String contactPoints;
+    private String username;
+    private String password;
+    private String tgtTable;
+    private String stTable;
+    private String expiryTable;
+    private String lastRunTable;
+
+    public String getContactPoints() {
+        return contactPoints;
+    }
+
+    public void setContactPoints(final String contactPoints) {
+        this.contactPoints = contactPoints;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(final String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(final String password) {
+        this.password = password;
+    }
+
+    public String getTgtTable() {
+        return tgtTable;
+    }
+
+    public void setTgtTable(final String tgtTable) {
```

Comment:
As you noted, this for instance shall go away because you could and should be able to use the ticket definition to pick up the table name.

Of course this sort of thing goes hand in hand with what I was pointing out earlier, which is that you'd then be able to create the table schemas automatically because everything is done automatically. Based on this design here, I think the auto-creation of tables at this point is sort of a must.